### DISCUSSION TITLE
Calendar module: input size

### DISCUSSION THREAD

**Discussion by cheesyMan** [fomantic/Fomantic-UI#1902] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
Is there a way to quickly manage the size of the **calendar** module input so it's of the same size of others inputs/dropdowns in the same form? As I see from the docs the calendar has no mini, tiny, small, large etc class to set

**Comment by lubber-de** [comment, marked as answer]
The calendar uses FUIs `input` element, which supports sizing. So try 

```html
<div class="ui calendar" id="standard_calendar">
  <div class="ui massive input left icon">
    <i class="calendar icon"></i>
    <input type="text" placeholder="Date/Time">
  </div>
</div>
```

See https://jsfiddle.net/lubber/umr19osn/

if you also want the calendar popup to be sized as well, add the desired sizing for the `table` component which is used inside the calendar module by changing the default for its table className like this:

```javascript
$('.ui.calendar')
  .calendar({
  	className:{
    	table: 'ui massive celled center aligned unstackable table',
    }
  })
;
```

See https://jsfiddle.net/lubber/umr19osn/3/

**Reply by cheesyMan** [reply]
thanks a lot
